x-eithert/src/X/Control/Monad/Trans/Either.hs

```diff
@@ -204,3 +211,28 @@ joinErrorsEither =
 tryEitherT :: (Functor m, MonadCatch m, Exception e) => (e -> x) -> m a -> EitherT x m a
 tryEitherT handler = firstEitherT handler . newEitherT . try
 {-# INLINE tryEitherT #-}
+
+-- | Lift an 'Either' into 'Errors'.
+hoistErrors :: Either e a -> Errors e a
```

Comment:
This name doesn't really match the naming / usage of the other two hoist functions:
```hs
hoistEither :: Monad m => Either x a -> EitherT x m a
hoistMaybe :: Monad m => x -> Maybe a -> EitherT x m a
```